apps/app/src/components/decisions/ResultsList.tsx

```diff
@@ -0,0 +1,93 @@
+'use client';
+
+import { trpc } from '@op/api/client';
+import { Header3 } from '@op/ui/Header';
+
+import { useTranslations } from '@/lib/i18n';
+
+import { EmptyProposalsState } from './EmptyProposalsState';
+import {
+  ProposalCard,
+  ProposalCardContent,
+  ProposalCardDescription,
+  ProposalCardFooter,
+  ProposalCardHeader,
+  ProposalCardMeta,
+} from './ProposalCard';
+
+const NoProposalsFound = () => {
+  const t = useTranslations();
+  return (
+    <EmptyProposalsState>
+      <Header3 className="font-serif !text-title-base font-light text-neutral-black">
+        {t('No results yet for this decision.')}
+      </Header3>
+      <p className="text-base text-neutral-charcoal">
+        {t('Results are still being worked on.')}
+      </p>
+    </EmptyProposalsState>
+  );
+};
+
+export const ResultsList = ({
+  slug,
+  instanceId,
+}: {
+  slug: string;
+  instanceId: string;
+}) => {
+  const t = useTranslations();
+
+  const [[instanceResults]] = trpc.useSuspenseQueries((t) => [
+    t.decision.getInstanceResults({
+      instanceId,
+    }),
+  ]);
```

Comment:
Should this be using `useSuspenseQuery`?

```suggestion
  const [instanceResults] = trpc.decision.getInstanceResults.useSuspenseQuery({
    instanceId,
  });
```